lib/main_micro_alp.pm

```diff
@@ -236,7 +236,7 @@ sub load_qemu_tests {
     loadtest 'microos/rebuild_initrd' if is_s390x;
     loadtest 'qemu/info';
     loadtest 'qemu/qemu' unless is_rt;
-    loadtest 'qemu/kvm' unless (is_aarch64 or is_rt);
+    loadtest 'qemu/kvm' unless (is_aarch64 or is_ppc64le or is_rt);
```

Comment:
@guoxuguang Current ppc test runs on a complete simulated ppc 'host' (in a vm) without kvm, right? Then launching kvm test seems not quite reasonable. We shall skip it IMO.

Reply:
Yes, based on the current situation, we'd better to skip kvm.